tests/unit/commands/setup_test.py

```diff
@@ -318,3 +318,8 @@ def test_get_boot2docker_vm_size_less_8_n(self, fake_virtual_memory, fake_get_ra
         self.assertEqual(_get_boot2docker_vm_size(), 1)
         fake_get_raw_input.assert_has_calls([call('Your system seems to have 6144 megabytes of memory. We would like to allocate 2048 to your vm. Is that ok? (y/n) '),
                                              call('Please input the number of megabytes to allocate to the vm: ')])
+    @patch('dusty.commands.setup._get_raw_input')
+    def test_enter_is_accepted_as_yes(self, fake_get_raw_input):
+        fake_get_raw_input.return_value = ''
+        setup_dusty_config()
```

Comment:
Shouldn't you do asserts at the end of this to confirm it did what you think it should do?